backend/src/test/java/com/funeat/acceptance/auth/AuthAcceptanceTest.java

```diff
@@ -86,16 +77,12 @@ class logout_성공_테스트 {
 
         @Test
         void 로그아웃을_하다() {
-            // given
-            final var loginCookie = 로그인_쿠키를_얻는다();
-            final var expected = "/";
-
-            // when
-            final var response = 로그아웃_요청(loginCookie);
+            // given && when
+            final var response = 로그아웃_요청(로그인_쿠키_획득(1L));
```

Comment:
```suggestion
            final var 사용자 = 로그인_쿠키_획득(1L);
            final var 응답 = 로그아웃_요청(사용자);
```
로그인을 해야 로그아웃을 할 수 있으니
이런식으로 흐름을 한번 끊어주는건 어떨까요?